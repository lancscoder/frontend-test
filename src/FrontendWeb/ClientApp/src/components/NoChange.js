import React from 'react';
import Glyph from './Glyph';

const NoChange = () => (
  <Glyph name="glyphicon glyphicon-minus icon-no-change"/>
);

export default NoChange;