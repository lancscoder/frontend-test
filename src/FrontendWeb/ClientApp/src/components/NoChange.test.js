import React from 'react';
import renderer from 'react-test-renderer';
import NoChange from './NoChange';

describe('NoChange ::', () => {
    it('should render with correct class', () => {
        const component = renderer.create(<NoChange />);;

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

        expect(tree.props.className).toBe('glyphicon glyphicon-minus icon-no-change');
    });
});