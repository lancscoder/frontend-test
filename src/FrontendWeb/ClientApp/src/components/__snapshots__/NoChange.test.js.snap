// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoChange :: should render with correct class 1`] = `
<span
  aria-hidden="true"
  className="glyphicon glyphicon-minus icon-no-change"
  style={
    Object {
      "paddingLeft": "5px",
      "paddingRight": "5px",
    }
  }
/>
`;
